test/test.js

```diff
@@ -814,4 +814,26 @@ describe('TreeModel', function () {
       });
     });
   });
+  
+  describe('with custom node class', function () {
+    var treeModel;
+
+    it('should throw an error when node class is invalid', function () {
+      function nodeClass() {}
+
+      assert.throws(function(){new TreeModel({nodeClass: nodeClass})}, TypeError,
```

Comment:
1. The string is the message that is displayed when the assertion fails, not asserting what the message should be. So just remove it.
2. IMO this is more readable:
```js
      assert.throws(
          function () {
            new TreeModel({nodeClass: nodeClass})
          },
          TypeError
      );
```